Extensions/TweenBehavior/tweenruntimebehavior.js

```diff
@@ -956,7 +956,10 @@ gdjs.TweenRuntimeBehavior.gdjsCallbackRuntimeSceneUnloaded = function(
 ) {
   if (!runtimeScene.shiftyJsScene) return;
 
-  runtimeScene.shiftyJsScene.stop(false); // TODO: check if necessary
+  runtimeScene.shiftyJsScene.stop(false);
+  runtimeScene.shiftyJsScene.tweenables.forEach(
+    runtimeScene.shiftyJsScene.remove.bind(runtimeScene.shiftyJsScene)
+  );
```

Comment:
@jeremyckahn @Wend1go I pushed the proper removal of tweens when a scene is unloaded. Seems not to improved the situation about other tweens being played though.. will try to investigate more.

Reply:
Dang. It's not immediately obvious to me what the issue might be. If you can give me steps to reproduce the issue (keeping in mind that I haven't really used GDevelop before) I can try to debug this over the weekend, if you'd like.

Reply:
I've quickly looked at the implementation of scenes and Shifty but can't see anything wrong 🙃Must be some silly mistake in GD.